_posts/2022-06-22-room-ride-sharing.md

```diff
@@ -0,0 +1,20 @@
+---
+author: DjangoCon US Organizers
+category: General
+date: 2022-06-22 13:00:00
+layout: post
+title: "Looking to share a room or ride to DjangoCon US 2022?"
+---
+🚗🏨 Are you looking to share lodging or transportation in San Diego?
+
+We have a <a href="https://docs.google.com/spreadsheets/d/1fSLtE122ZsImD006aOs45-Q4Asd9hAg8-ZLAUoLCajo/edit?usp=sharing">spreadsheet</a> where you can fill out information to share lodging and/or transportation.
+
+## What are my options?
```

Comment:
Do we want to add any mention of the Code of Conduct or that people should be careful about getting in a car with someone they don't know / sleeping in their hotel room?

Reply:
Great ideas

Reply:
Yeah it seems okay, @jefftriplett any thoughts?

Reply:
@drewbrew  @logankilpatrick  The second one is bugging me and I am having trouble pinning down why. I think it comes across as a little paternalistic or victim-blamey, with "Take care of yourself." and "Safety first" to me implies that we shouldn't even offer this at all because we can't guarantee it's safe (which is not me suggesting we not offer this). I think just reminding that the Code of Conduct also applies to arrangements made from the sharing sheet is sufficient.

Reply:
I agree. I'm going to yank that bit entirely.

Reply:
Done!